app/scripts/modules/core/src/managed/Environments.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export default function Environments() {
+  return <div>For there shall be no greater pursuit than that towards desired state.</div>;
```

Comment:
![beautiful](https://media.giphy.com/media/it6W8D4FfvaPC/giphy.gif)

Reply:
> we should figure out a strategy for when to surface this in the nav
Yes, my thoughts exactly. I imagine it may also evolve as MD matures.